common_features.mk

```diff
@@ -114,6 +114,13 @@ else
         SRC += $(PLATFORM_COMMON_DIR)/flash_stm32.c
         OPT_DEFS += -DEEPROM_EMU_STM32F072xB
         OPT_DEFS += -DSTM32_EEPROM_ENABLE
+      else ifeq ($(MCU_SERIES)_$(MCU_LDSCRIPT), STM32F0xx_STM32F042x6)
+        USE_PROCESS_STACKSIZE = 0x600
+        USE_EXCEPTIONS_STACKSIZE = 0x300
```

Comment:
I'd prefer these moved out of the EEPROM area -- they're unrelated items and should probably go to `quantum/mcu_selection.mk` instead.

Reply:
Yes, that looks like a better place for it.

Reply:
What if someone decides to use an external EEPROM chip?

Reply:
Then, you could set `EEPROM_DRIVER` to something other than `vendor`, right? The behaviour of including EEPROM emulation by default is the same for all STM32 chips.

Reply:
Yes, but the stack will be shrunk regardless. Is that intentional? (ie. when using external EEPROM, we still need to store the page buffer in RAM?)

Reply:
That is true. I cannot check for the currently used driver at this stage, because it is set later in `common_features.mk`. Do you have a suggestion?

Reply:
If it's specifically to deal with flash-based EEPROM requirements, then I'd say keep it in the original location you had but document (i.e. with inline comments) why the overrides are present and why those specific values were chosen, (Which you've done, great!)

As it seems these were added in order to deal with constraints specific to when EEPROM emulation is enabled, then it makes sense to keep those configurations localised to the area that requires them -- the point is to ensure that if someone decides they need to make modifications in 12 months time... then we have rationale as to why it's present and for what cases it's required, with a much lower likelihood of them being moved or removed.

Obviously we'd prefer not to mistakenly carry it over to all implementations, like my invalid assumption and subsequent request to move the configuration would've caused as I didn't initially understand the rationale. I'm happy with it in its current location, as it's properly documented.

Reply:
Great, thanks for your comment! Indeed, an explanation for reducing the stack size was previously missing and it was important to add it. The stack size could also be defined in the user application (`rules.mk`) but that is probably not ideal either.